packages/components/text-field/src/text-field.stories.ts

```diff
@@ -15,7 +15,6 @@ const sizes: TextFieldSize[] = ['md', 'lg'];
 export const API: StoryObj = {
```

Comment:
If we want to keep those form stories, could you please update text fields there as well? :)

<img width="787" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/82797062-5f0e-481e-926d-f115a4af77e8">

Reply:
I'll refactor the "Forms" stories later (as I mentioned in the checkbox PR).